.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,28 @@
+name: Lint python code
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: '**'
+
+jobs:
+  pylint:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout code including full history and submodules
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
```

Comment:
Neither history nor submodules are needed for this.

Reply:
True, will remove

Reply:
ok will remove